btax/run_btax.py

```diff
@@ -46,7 +46,7 @@
 ASSET_PRE_CACHE_FILE = 'asset_data.pkl'
 
 
-def run_btax(test_run,baseline=False,start_year=2016,iit_reform=None,**user_params):
+def run_btax(test_run,baseline=False,start_year=2017,iit_reform=None,**user_params):
```

Comment:
@jdebacker  Looks like we have a few functions with a signature involving `start_year=2017`.  Let's move `2017` to a constant in one place, such a util module, that can be imported into the modules needing this type of signature, changing the signature to `start_year=DEFAULT_START_YEAR`.